File: app/src/main/java/com/example/android/ecommerce/model/Tax.java
package com.example.android.ecommerce.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Tax {

    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("value")
    @Expose
    private long value;

    /**
     * No args constructor for use in serialization
     */
    public Tax() {
    }

    /**
     * @param name
     * @param value
     */
    public Tax(String name, long value) {
        super();
        this.name = name;
        this.value = value;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public long getValue() {
        return value;
    }

    public void setValue(long value) {
        this.value = value;
    }
}

File: app/src/main/java/com/example/android/ecommerce/model/Tax.kt
package com.example.android.ecommerce.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Tax(@SerializedName("name")
               var name: String = "",
               @SerializedName("value")
               var value: Double = 0.0) : Parcelable